src/nasuta-sebastian/Fsd.Sebastian.Client/Fsd.Sebastian.Client.Ex2/quiz.js

```diff
@@ -0,0 +1,63 @@
+var app = app || {};
+
+(function () {
+    const COUNT_ATTRIBUTE = 'question-count';
+    const COUNTER = document.getElementById('counter');
+    const QUESTIONS_AMOUNT = 3;
+    var questions = [];
+    var answers = [];
+
+    function initialize() {
+        var xhttp = new XMLHttpRequest();
```

Comment:
to warto wydzielić do osobnego komponentu odpowiedzialnego za czytanie np. 'ajaxReader' itp (teraz quiz.js nie spełnia SOLID)